projects/netgrif-components-core/src/lib/panel/task-panel/abstract-task-panel.component.ts

```diff
@@ -342,6 +344,10 @@ export abstract class AbstractTaskPanelComponent extends AbstractPanelWithImmedi
             ? this.taskPanelData.task.finishTitle : 'tasks.view.finish';
     }
 
+    get overflowMode(): boolean {
```

Comment:
`AbstractTaskPanelComponent` and `AbstractCasePanelComponent` are extending `AbstractPanelWithImmediateDataComponent`, that extends `AbstractPanelWithHeaderBindingComponent`. Would not be a good option to extract this method to `AbstractPanelWithHeaderBindingComponent`? Duplicates can be avoided by this.

Reply:
Done